src/main/java/org/jitsi/videobridge/WebRtcDataStream.java

```diff
@@ -160,7 +189,19 @@ public void sendString(String strMsg)
     public void onBinaryMsg(byte[] binMsg)
     {
         if(dataCallback != null)
+        {
             dataCallback.onBinaryData(this, binMsg);
+        }
+        else
+        {
+            // NOTE consider adding analytics for the missed out data to detect
+            // bugs ?
+            logger.error(
```

Comment:
Just out of curiosity, has this ever occurred during your trials? If so, have you been able to identify the conditions under which this may occur?